src/integration_tests/datasets/e2e_dataset_integration_test.py

```diff
@@ -45,7 +45,12 @@ def test_dataset_e2e(self):
         if create_dataset_response is not None and create_dataset_response != 200:
             assert False, "Unsuccessful request to create dataset"
 
-        assert not get_dataset_bucket().blob(filename).exists()
+        assert (
```

Comment:
Not sure why do we have to change the way to get dataset bucket

Reply:
because that method no longer exists. It was tieing our testing implementation to our src code which isnt best to do and was causing issues

Reply:
Ok. So do we have to remove the function from bucket_helpers entirely, and do the same for all places using bucket_loader.get_dataset_bucket() and bucket_load.get_schema_bucket() as well? They are still being used in the cleanup method